contrastsecurity/datadog_checks/contrastsecurity/data/conf.yaml.example

```diff
@@ -0,0 +1,14 @@
+init_config:
+
+instances:
+
+  -
+
+    ## @param tags - list of key:value elements - optional
```

Comment:
Remove this, it does not make sense since there are no metrics

Reply:
I did not mean remove the file, but the tags section you need the file to hold the logs config.